docs/config.json

```diff
@@ -31,27 +31,39 @@
             {
               "title": "Teleport Cloud Agents (Linux)",
               "slug": "/upgrading/cloud-linux/",
-              "forScopes": ["cloud"]
+              "forScopes": [
```

Comment:
It would be a lot easier to review the change if there weren't unrelated formatting changes. Looks like you might have some sort of auto-formatter configured?

Reply:
hmm not intentionally, I'm on Visual studio code 1.89.0 (Universal). I'll see if I can disable it.